### ISSUE TITLE
Automatically activate?

### ISSUE THREAD

**Issue by mp3guy** [nathanielsimard/overview-navigation#45]
Is there a way to automatically active this extension every time without having to trigger it with the space bar?

**Comment by nathanielsimard** [comment]
Yes in the settings, but I think they are broken in the version from gnome.org. Feel free to build from master.
